Formula/snyk.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Snyk < Formula
+  desc "CLI and build-time tool to find & fix known vulnerabilities in open-source dependencies"
+  homepage "https://snyk.io"
+  url "https://registry.npmjs.org/snyk/-/snyk-1.136.1.tgz"
+  version "1.136.1"
+  sha256 "05a347ae082533f50e7132fc86370b244a910f6d430d5747d63423a3a451feaa"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"snyk", "version"
```

Comment:
Could you try and do a more meaningful test then this? Version checks rarely give a real indication of a failing build.

Reply:
Thanks, pushing the change!

Reply:
Got delayed a bit, will probably get back to it tomorrow, or next week.
Really appreciate the responsiveness here :)